src/shared/agent_op.c

```diff
@@ -619,8 +622,11 @@ static cJSON* w_create_agent_add_payload(const char *name,
         cJSON_AddStringToObject(arguments, "id", id);
     }
 
-    if (force >= 0) {
-        cJSON_AddNumberToObject(arguments, "force", force);
+    if (force_options) {
```

Comment:
If **w_force_options_to_json()** will return NULL in case of a NULL force_options, we can call **w_force_options_to_json()** directly, without checking **force_options** but cheking the returned **j_force**.

Reply:
You are right !